pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -71,6 +84,11 @@ const TagNameSubnetInternalELB = "kubernetes.io/role/internal-elb"
 // it should be used for internet ELBs
 const TagNameSubnetPublicELB = "kubernetes.io/role/elb"
 
+// ServiceAnnotationLoadBalancerType is the annotation used on the service
+// to indicate what type of Load Balancer we want. Right now, the only accepted
+// value is "nlb"
```

Comment:
It might be nice to allow someone to explicitly specify "elb" in future, but not a big deal